site2/docs/tutorials-topic.md

```diff
@@ -0,0 +1,34 @@
+---
+Id: tutorials-topic
+title: How to create a topic
+sidebar_label: “Tutorials”

```

Comment:
The `sidebar_label` parameter refers to the display text in the left navigation. 
All these `How-To` topics have the same sidebar label and the `How-to-landing` topic hasn't got one, which might be a potential issue if you want to display them in the left navigation later.

Reply:
I don't want to display those in left navigation. The landing page is the navigation for those topics.